README.md

```diff
@@ -42,6 +42,18 @@ environment.
 6. For more information on setting up Word add-in for development, follow this
    [documentation](https://docs.microsoft.com/en-us/office/dev/add-ins/quickstarts/word-quickstart).
 
+#### Debugging Add-in
+-  If you're on the web version of Word, you can simply open the developer tools
+   and see the console's output.
+
+If you are on the desktop version,
+
+-  **Windows:** You will need to
+   [download the developer tool](https://www.microsoft.com/en-us/p/microsoft-edge-devtools-preview/9mzbfrmz0mnj?activetab=pivot:overviewtab)
+   separately. Once you are able to start the server and sideload the add-in,
+   you will see "Accord Project" as an option in the tool.
+   ![edge developer tool](readme_assets/devtool.png)
+
```

Comment:
A few suggestions: 

```
#### Debugging Add-In
- Web version: ...
- Desktop version:
  - **Windows**: ...
  - **MacOS**: ...
```

Also avoid "simply" in the spirit of [inclusive documentation](https://developers.google.com/style/inclusive-documentation)